student.rb

```diff
@@ -0,0 +1,11 @@
+class Student < Person
+  attattr_accessor :@classroom
```

Comment:
- [ ] Kinldy make sure the 
```rb 
attr_accessor # is well written. not
attattr_accessor
```
- [ ] and as well the `instance variable` shouldn't written like that on the `getter`
```suggestion
  attr_accessor :classroom
```